### DISCUSSION TITLE
How do I create an ICoreWebView2EnvironmentOptions, in a language other than C++, such as Pascal or VB?

### DISCUSSION THREAD

**Discussion by GerbenAbbink** [MicrosoftEdge/WebView2Feedback#3905] [Q&A]
In C++ I see this:

Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>()

to create an ICoreWebView2EnvironmentOptions.

How to I do that in a different language, such as Pascal or VB?

**Comment by champnic** [comment]
Hey @GerbenAbbink - For VB you can use VB.NET and our .NET controls. This getting started guide uses C# but should be similar in VB. You can search this forum for examples of others using VB:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winforms

For Pascal, I have no idea unfortunately.

**Comment by GerbenAbbink** [comment]
To be more precise:

I am trying to create an embedded WebView2 with argument "--allow-file-access-from-files" by calling CreateCoreWebView2EnvironmentWithOptions(). But I have no idea how to create ICoreWebView2EnvironmentOptions in Pascal.

**Comment by amancini** [comment]
For pascal tour can try https://github.com/amancini/AMEdgeBrowser
